include/deal.II/numerics/data_out.h

```diff
@@ -166,8 +166,35 @@ class DataOut : public DataOut_DoFData<DoFHandlerType, DoFHandlerType::dimension
   active_cell_iterator;
 
   /**
-   * Enumeration describing the region of the domain in which curved cells
-   * shall be created.
+   * Enumeration describing the part of the domain in which cells
+   * should be written with curved boundaries. In reality, no file
+   * format we are aware of really supports curved boundaries, but
+   * this can be emulated by plotting edges as a sequence of straight
+   * lines (and faces in 3d as a collection of bilinear patches) if
+   * DataOut::build_patches() is called with a number of subdivisions
+   * greater than 1.
+   *
+   * The elements of this enumeration then describe for which cells
+   * DataOut::build_patches() will query the manifold or boundary
+   * description for curved geometries:
+   *
+   * - @p no_curved_cells: The geometry or boundary description
```

Comment:
Yes. Right now they are undocumented, so they won't even show up in the documentation.